snapshots/ilostat/2023-09-19/unemployment.csv.dvc

```diff
@@ -0,0 +1,33 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Education and Mismatch Indicators (EMI) - Unemployment
+    description: |-
+      With the aim of promoting international comparability, statistics presented on ILOSTAT are based on standard international definitions wherever feasible and may differ from official national figures. This series is based on the 13th ICLS definitions. For time series comparability, it includes countries that have implemented the 19th ICLS standards, for which data are also available in the Work Statistics -- 19th ICLS (WORK) database. The unemployment rate conveys the number of persons who are unemployed as a percent of the labour force (i.e., the employed plus the unemployed). The unemployed comprise all persons of working age who were: a) without work during the reference period, i.e. were not in paid employment or self-employment; b) currently available for work, i.e. were available for paid employment or self-employment during the reference period; and c) seeking work, i.e. had taken specific steps in a specified recent period to seek paid employment or self-employment. Data disaggregated by level of education are provided on the highest level of education completed, classified according to the International Standard Classification of Education (ISCED). Data may have been regrouped from national classifications, which may not be strictly compatible with ISCED. For more information, refer to the Education and Mismatch Indicators (EMI) database description.
+
+      Classification details	Data by age includes various age bands, usually including but not limited to youth (ages 15-24), prime-age workers (age 25-54), and older workers (ages 55+). Data by sex includes data on both sexes (total), men (male) and women (female). Educational attainment based on the International Standard Classification of Education (ISCED). Categories include less than basic (less than primary education), basic (primary and lower secondary education), intermediate (upper secondary and post-secondary non-tertiary education) and advanced (short-cycle tertiary education, bachelor's, master's and doctoral or equivalent levels).
+    date_published: latest
```

Comment:
Why "latest"? This should be the date when the current version of the data product was published.